workflows/phaseimpute/main.nf

```diff
@@ -189,7 +190,7 @@ workflow PHASEIMPUTE {
         // Posfile
         exportCsv(
             ch_posfile.map{ meta, vcf, index, hap, legend ->
-                [meta, [2:"prep_panel/sites/vcf", 3:"prep_panel/haplegend", 4:"prep_panel/haplegend"], vcf, index, hap, legend]
+                [meta, [2:"prep_panel/sites", 3:"prep_panel/haplegend", 4:"prep_panel/haplegend"], vcf, index, hap, legend]
```

Comment:
Just to be sure, if the panel is normalized are they still going to be written here `prep_panel/sites` rather than `prep_panel/sites/vcf?`

Reply:
Yes indeed, it is visible in the snapshot.